cowprotocol/composable-cow
# Remove redundant deployment information

# Description

The `broadcast` folder contains (almost duplicated files). This PR removes one of them.

# Changes

- Remove any deployment run that isn't `-latest`.

## How to test

To verify that it's indeed redundant, run on `main`:

<details><summary>for chain in 1 5 100; do echo "Diff chain $chain:"; diff "$chain/"*; done</summary>

```
$ for chain in 1 5 100; do echo "Diff chain $chain:"; diff "$chain/"*; done
Diff chain 1:
166,197d165
<       "transactionHash": "0x33dcbc73a8797c69a5b3956539dd8d191cf3f190bcb27a4d4eca8556f030f574",
<       "transactionIndex": "0x9",
<       "blockHash": "0x6e1e48bd1da75b8015d48dbd1ebf9105a7f02f7e13fdba0fd7793ed1f17f2183",
<       "blockNumber": "0x110df9b",
<       "from": "0x72716d1e51BcC48819A3Ce57F6cD81B938a67987",
<       "to": "0x4e59b44847b379578588920cA78FbF26c0B4956C",
<       "cumulativeGasUsed": "0x1f9d12",
<       "gasUsed": "0x1f9d12",
<       "contractAddress": "0x2f55e8b20d0b9fefa187aa7d00b6cbe563605bf5",
<       "logs": [],
<       "status": "0x1",
<       "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
<       "type": "0x2",
<       "effectiveGasPrice": "0x4459a5ab1"
<     },
<     {
<       "transactionHash": "0x2d4159ae0d4f026e2063292f3e40d898982619853484b8cd84f3748503fcdd19",
<       "transactionIndex": "0x11",
<       "blockHash": "0x393c6d7bc81249ef886f48121a7b88878c20ec0e5a5c6eb230e36c2e787975f6",
<       "blockNumber": "0x110dfa9",
<       "from": "0x72716d1e51BcC48819A3Ce57F6cD81B938a67987",
<       "to": "0x4e59b44847b379578588920cA78FbF26c0B4956C",
<       "cumulativeGasUsed": "0x1ccda5",
<       "gasUsed": "0x1ccda5",
<       "contractAddress": "0xfdafc9d1902f4e0b84f65f49f244b32b31013b74",
<       "logs": [],
<       "status": "0x1",
<       "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
<       "type": "0x2",
<       "effectiveGasPrice": "0x40acf86ea"
<     },
<     {
Diff chain 5:
Diff chain 100:
```
</details>

Notice also that the mainnet file we kept is the one with more information.
